GamingAnthems/dist/metadata.json

```diff
@@ -13,8 +13,8 @@
     "raidstop.com"
   ],
   "version": "1.0.0",
-  "logo": "https://imgur.com/9PEupja",
-  "thumbnail": "https://imgur.com/j7u9H9X",
+  "logo": "https://i.imgur.com/9PEupja.png",
+  "thumbnail": "https://i.imgur.com/j7u9H9X.png",
   "color": "#000000",

```

Comment:
sorry I only just noticed it, but the thumbnail should be an image of the website, if you dont have an actual thumbnail.

Reply:
??

Reply:
it can be a promotional image as well